identityClient/identityClient.go

```diff
@@ -1346,3 +1346,14 @@ func (c *E3dbIdentityClient) UpdatePAMJiraPlugin(ctx context.Context, params Upd
 	err = e3dbClients.MakeSignedServiceCall(ctx, c.requester, request, c.SigningKeys, c.ClientID, &plugin)
 	return plugin, err
 }
+
+// TestPAMJiraPlugin tests a Jira integration's credentials
```

Comment:
how is the test performed / what does success mean?

Reply:
updated comment 👍